frontend/src/app/main/data/workspace/common.cljs

```diff
@@ -372,6 +371,12 @@
                     #{}
                     ids)
 
+            interacting-shapes
+            (filter (fn [shape]
+                      (let [interactions (:interactions shape)]
+                        (seq (filter #(contains? ids (:destination %)) interactions))))
```

Comment:
I think `some` is a better idiom here (instead of filter + seq)

Reply:
And if  ids is a **set**, this can expressed as `(some ids (map :destination interactions))` if I'm not wrong

Reply:
good trick, very compact!